File: backend/src/admin/dto/log.pagenation.dto.ts
import { ApiProperty } from '@nestjs/swagger';
import { CabinetLentLogDto } from './cabinet-lent-log.dto';

export class LogPagenationDto {
  @ApiProperty({
    description: '로그 배열',
    type: [CabinetLentLogDto],
  })
  result: CabinetLentLogDto[]; // 대여 정보

  @ApiProperty({
    description: 'DB에 저장된 총 결과의 길이',
    example: 42,
  })
  total_length: number; // DB에 저장된 총 결과의 길이
}

File: backend/src/main/java/org/ftclub/cabinet/dto/LogPageNationDto.java
package org.ftclub.cabinet.dto;

import lombok.*
@AllArgsConstructor
@NoArgsConstructor
@Data
public class LogPagenationDto {
    /**
     * NOTE:
     * 이름의 통일성을 위해 CabinetLentLogPagenationDto로 수정하는 것이 어떤지?
     */
    private CabinetLentLogDto[] result;
    private Long totalLength;
}